rasa/core/policies/memoization.py

```diff
@@ -236,6 +236,16 @@ def predict_action_probabilities(
 
         return result
 
+    def format_tracker_states(self, states):
+        # Format tracker states to human readable format on debug log
+        formatted_states = ''
+        for index, state in enumerate(states):
+            if bool(state):
```

Comment:
Is this check necessary? Is it possible for the state to be None or an empty dict? If so, I think you don't need to cast it to bool, you can just check `if state:`